.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:
+
+- [ ] Branch rebased on latest master `git pull --rebase upstream master`
+- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
+- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
+- [ ] Commit message includes `Fixes #<id>` where `<id>` is the associated MantisBT issue id if one exists
+
+And please remember to update the Wiki with the features added or changed once this PR once it is merged.
```

Comment:
Line 8: `And please remember to update the Wiki with the features added or changed once this **PR is merged**.`